src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -287,8 +665,18 @@ public void binaryContentNoTypeExternalType(final String uri) {
     public void binaryContentNoTypeDefault(final String uri) {
         final TestInfo info = setupTest("3.9-F-3", "binaryContentNoTypeDefault",
                 "Fedora servers must use the value of the type attribute in the external content link as the media " +
-                        "type of the external content, if provided. Servers may use a default media type.",
+                        "type of the external content, if provided. If there is no type attribute: " +
+                        "Servers may use a default media type.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUriWithNoType, "proxy")),
+                new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.